lib/matplotlib/offsetbox.py

```diff
@@ -187,6 +187,14 @@ def set_figure(self, fig):
         for c in self.get_children():
             c.set_figure(fig)
 
+    @martist.Artist.axes.setter
+    def axes(self, ax):
+        # TODO deal with this better
+        martist.Artist.axes.fset(self, ax)
```

Comment:
This is how you call the base-class property setter function.

https://docs.python.org/2/howto/descriptor.html#properties